changelog/generate_rst.py

```diff
@@ -217,13 +218,14 @@ def _render_rec(changelog_directive, rec, section, cat, append_sec):
                 node = nodes.Text(prefix % refname, prefix % refname)
             insert_ticket.append(node)
 
-    if rec["tags"]:
+    tags = rec["tags"] or {'no_tags'}
```

Comment:
IIUC this will print the tag "no_tags" if there are none, right?

Reply:
Yes, because otherwise the changelog item will have no caption, just empty line with a link icon which is hidden if not hovered by mouse pointer. That's why I've added some text here.